### PR TITLE
Branch of #1382 with new strategy

### PR DESCRIPTION
**Pull request by marcharper**

### CODE REVIEW

**Review by Nikoleta-v3 — APPROVED** [review]

**Comment by drvinceknight** [comment]
Looks good to me, will wait for @dashiellfryer to give ok as this will make #1382 redundant (if I'm not mistaken?).

**Comment by marcharper** [comment]
Docstring updated, ready to go in I think. I also verified that this strategy is the new winner of the short run time tournament.

File: axelrod/strategies/finite_state_machines.py [hunk 1]
```diff
@@ -1024,3 +1024,48 @@ def __init__(self) -> None:
...
             transitions=transitions, initial_state=0, initial_action=C
         )
+
+
+class EvolvedFSM6(FSMPlayer):
+    """
+    An 6 state FSM player trained with an evolutionary algorithm.
+
+    Evolved using axelrod-dojo version 0.0.8 and axelrod version 4.10.0, trained to maximize score against
+    the short_run_time_strategies with 10 machine states for 500 generations,population size of 40,
```
Comment (Nikoleta-v3) [inline axelrod/strategies/finite_state_machines.py]:
```suggestion
    the short_run_time_strategies with 10 machine states for 500 generations, population size of 40,
```

**Comment by drvinceknight** [comment]
> Docstring updated, ready to go in I think. I also verified that this strategy is the new winner of the short run time tournament.

Nice! Thanks for the contribution @dashiellfryer :+1:, will do a new release and rerun the tournament repo :+1: :D

**Pull request merged**